lib/workers/repository/error.js

```diff
@@ -1,13 +1,15 @@
-const { raiseConfigWarningIssue } = require('./error-config');
+import { logger, setMeta } from '../../logger';
+import { raiseConfigWarningIssue } from './error-config';
 
-module.exports = {
+export default {
   handleError,
 };
 
 async function handleError(config, err) {
```

Comment:
and /or: `export async function handleError(config, err)`

then we can use it like `const { handleError } = require('./error')`